config/rbac/role.yaml

```diff
@@ -2,7 +2,6 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  creationTimestamp: null
```

Comment:
is this change needed?

Reply:
see https://github.com/openshift/windows-machine-config-operator/pull/2405#discussion_r1734821011